src/app/static/preview-question/preview-question.component.ts

```diff
@@ -24,30 +24,32 @@ export class PreviewQuestionComponent implements OnInit, AfterViewChecked {
   @Input() question: Question;
   @Input() tag: Tag;
 
+  @Output() change: EventEmitter<number> = new EventEmitter<number>();
+
```

Comment:
~Please name this something appropriate to what it does.  What does the number it emits represent?~
As mentioned prior, this functionality will probably be reworked, so nevermind!